deploy/operator.yaml

```diff
@@ -26,6 +26,8 @@ spec:
           value: ""
         - name: IMAGE_ADDONS_PARAM_KN_IMAGE
           value: registry.redhat.io/openshift-serverless-1/client-kn-rhel8:0.13.2
+        - name: IMAGE_ADDONS_SKOPEO_COPY
+          value: registry.redhat.io/rhel8/skopeo
```

Comment:
naive question : how is the *magically* used ? :stuck_out_tongue_closed_eyes:

Reply:
:stuck_out_tongue_closed_eyes: it took some time for me to get it, 
but the name of the env variable is the main magic and else is taken care [here](https://github.com/openshift/tektoncd-pipeline-operator/blob/2f271f6761de79fb494c4c8a570486f6e1c28217/pkg/utils/transform/transform.go#L277) and then [here](https://github.com/openshift/tektoncd-pipeline-operator/blob/2f271f6761de79fb494c4c8a570486f6e1c28217/pkg/utils/transform/transform.go#L312)